### PR TITLE
Updated UI, and resolves bugs and checkstyles.

### PR DESCRIPTION
**Pull request by LilideBree**
-This updated the coins, #259.
-Changed the Bubs, so they are both the same shape, and now have different colours to each other when they die. There is also added death images, so bub can face either way when he dies, #257.
-Removed a FindBug error and removed copied code, #279 
-Added Javadoc to the model classes, #278

### CODE REVIEW

File: src/main/java/model/Player.java [hunk 1]
```diff
@@ -418,7 +424,12 @@ public void checkCollideMonster(final Monster monster) {
     public void die() {
         this.loseLife();
         this.scorePoints(Settings.POINTS_PLAYER_DIE);
-        spriteBase.setImage("Bub" + playerNumber + "Death.png");
+        if (isFacingRight) {
+            spriteBase.setImage("Bub" + playerNumber + "RightDeath.png");
+        }
+        if (!isFacingRight) {
+        	spriteBase.setImage("Bub" + playerNumber + "LeftDeath.png");
```
Comment (Verzijl) [inline src/main/java/model/Player.java]: Please make sure the indentation is right. Makes it better readable.

File: src/test/java/model/PlayerTest.java [hunk 2]
```diff
@@ -164,7 +164,8 @@ public void testCheckCollideMonster() throws Exception {
                 sprite.getY() + sprite.getHeight())).thenReturn(true);
         player.checkCollideMonster(monster);
         assertTrue(player.isDead());
-    }
+    } 
+
```
Comment (Verzijl) [inline src/test/java/model/PlayerTest.java:168]: This is not necessary. Not useful for the diffs.

File: src/test/java/model/PlayerTest.java [hunk 3]
```diff
@@ -181,6 +182,7 @@ public void testDie() throws Exception {
         assertEquals(x, sprite.getX(), 0.001);
         assertEquals(0, sprite.getDx(), 0.001);
     }
+
```
Comment (Verzijl) [inline src/test/java/model/PlayerTest.java:184]: This is not necessary. Not useful for the diffs.

File: src/test/java/model/PlayerTest.java [hunk 4]
```diff
@@ -342,6 +344,7 @@ public void testTimerDead() {
         verify(spriteBase, atLeastOnce()).setImage(anyString());
     }
 
+
```
Comment (Verzijl) [inline src/test/java/model/PlayerTest.java:346]: This is not necessary. Not useful for the diffs.

**Comment by JPluim** [comment]
:shipit:

**Pull request merged**